core/src/syn/v2/lexer/keywords.rs

```diff
@@ -3,9 +3,54 @@ use crate::{
 	sql::{language::Language, Algorithm},
 	syn::v2::token::{DistanceKind, Keyword, TokenKind},
 };
-use phf::phf_map;
+use phf::{phf_map, phf_set};
 use unicase::UniCase;
 
+/// A set of keywords which might in some contexts are dissallowed as an identifier.
+pub static RESERVED_KEYWORD: phf::Set<UniCase<&'static str>> = phf_set! {
+	UniCase::ascii("ANALYZE"),
+	UniCase::ascii("BEGIN"),
+	UniCase::ascii("BREAK"),
+	UniCase::ascii("CANCEL"),
+	UniCase::ascii("COMMIT"),
+	UniCase::ascii("CONTINUE"),
+	UniCase::ascii("CREATE"),
+	UniCase::ascii("DEFINE"),
```

Comment:
So, I can see there is no "TABLE" or "DATABASE" which is great. Wondering if we also want the "INCLUDE". We already have "DIFF" and correctly don't have "ORIGINAL". The syntax would be 
`DEFINE TABLE test CHANGEFEED 10m INCLUDE DIFF`

The current codebase has examples of `INCLUDE ORIGINAL`

Also debating if we want DIFF or PATCH for the INCLUDE

Reply:
These keywords are only for keywords which can be disallowed. Dissallowed keywords happen when at a branch in the grammar a keyword and a identifier are both valid tokens of different branches. Take forinstance in the definition of the LiveStatement. The live statement is defined as following:
```
LiveStatement => 'LIVE' 'SELECT' LiveFields FROM etc...

LiveFields => 'DIFF'
LiveFields => [Lookahead =/= 'DIFF'] Value
```
The look-ahead disallows DIFF because otherwise the parser wouldn't be able to decide if `LIVE SELECT diff FROM bla` meant you want to select the identifier `diff` or you want the keyword diff so you have to write
```
LIVE SELECT `diff` FROM bla
```
if you want to do the ident option. 

This list is all the keywords which could in some cases be disallowed by a look-ahead clause. Identifiers which match these keywords need to be escaped because otherwise they might not parse correctly. 

As far as I am aware there isn't a place in the syntax where the token INCLUDE could both be parsed as an keyword and an identifier so I don't think adding INCLUDE is necessary.